tests/scripts/pulp_python/test_content.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+# shellcheck source=tests/scripts/config.source
+. "$(dirname "$(dirname "$(realpath "$0")")")"/config.source
+
+pulp debug has-plugin --name "python" --min-version "3.1.0.dev" || exit 3
```

Comment:
Do we really need to skip all those tests with versions less than 3.1?
If yes, this means the commands only start to work properly with 3.1. And in that case, you should require that version in `__init__.py`.

https://github.com/pulp/pulp-cli/pull/73/files#diff-607abf748c857c19b5f894cd196a77a9429d608fe1b80c2b666e45f909ed029fR13